server/src/test/java/com/google/coffeehouse/storagehandler/StorageHandlerTest.java

```diff
@@ -47,6 +50,139 @@ public void setUp() {
     StorageHandlerTestHelper.setUpClearDb();
   }
 
+  @Test
+  public void getPerson_doesNotExistInDb() throws Exception {
+    assertThrows(RuntimeException.class, () -> {
+      StorageHandler.getPerson(dbClient, "personNotInDb");
+    });
+  }
+
+  @Test
+  public void getPerson_existsWithPronouns() throws Exception {
+    StorageHandlerTestHelper.insertPersonWithPronouns("person");
+    Person actual = StorageHandler.getPerson(dbClient, "person");
+    Person expected = StorageHandlerTestHelper.createTestPersonObject(
+      "person", /**pronouns = **/true);
```

Comment:
super minor nit: Inline comments just use `/* ... */`. The double-asterisks are for JSDoc comments

Reply:
Thank you for pointing this out!